MSBuild.Sdk.Extras/Sdk/Sdk.targets

```diff
@@ -0,0 +1,9 @@
+<Project>
+
+  <PropertyGroup>
+    <MSBuildAllProjects>$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
+  </PropertyGroup>
+
+  <Import Project="$(MSBuildSDKExtrasTargets)" />
```

Comment:
Is there any reason to keep the variable defined in the props and imported here? Shouldn't we just import the targets directly? Or is that to preserve usage in both scenarios?

Reply:
Yeah, I wasn't sure which way you might want it to go. 

Not sure if it makes sense to keep the "old" usage in the readme even.

Reply:
I think we should probably keep it until 15.6 is stable since that's the min version that supports this. What would the corresponding VSfM version that supports this import style as well?

Then we can remove it in Extras 1.3.

Reply:
Yep, makes sense. Not sure about VSfM. @mhutch?